project.php

```diff
@@ -377,7 +377,8 @@ function do_blurb_box( $blurb )
     if ( is_null($blurb) ) return;
 
 //    echo "<br>";
-    echo "<table style='width: 100%; max-width: 630px; background-color: #DDDDDD;'>";
+    $class = 'blurb-box';
+    echo "<table class=$class>";
```

Comment:
This will mean the rendered HTML will be `<table class=blurb-box>` (no quotes). I would skip the variable and just put it directly into the echo.

Reply:
Thanks for catching that. Fixed. (I hope 😁 )